libcloud/compute/drivers/ec2.py

```diff
@@ -3630,7 +3630,7 @@ def ex_register_image(self, name, description=None, architecture=None,
                           image_location=None, root_device_name=None,
                           block_device_mapping=None, kernel_id=None,
                           ramdisk_id=None, virtualization_type=None,
-                          ena_support=None):
+                          ena_support=None, billing_product=None):
```

Comment:
I'm referring to this doc: http://docs.aws.amazon.com/cli/latest/reference/ec2/register-image.html

The BillingProduct.N notation is implying multiple as well.